pom.xml

```diff
@@ -231,6 +248,14 @@
                         </dependency>
                     </dependencies>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-scm-publish-plugin</artifactId>
```

Comment:
+1 on this plugin.  I had gotten around to this config at some point in my experimenting and think it's the way to go. The only thing I couldn't figure out was tying it to the lifecycle... but that's OK as I think we want it separate from the lifecycle anyway.